.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Report an issue or bug with the Quantum Core X.
+title: "bug: "
+labels: [ "type/bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help us diagnose and fix the issue. Before submitting a bug report, please check the [Reporting Bugs](https://github.com/MeikelLP/quantum-core-x/blob/master/CONTRIBUTING.md#reporting-bugs) section in the contributing guidelines.
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: A clear and concise description of what actually happened.
+      placeholder: Describe the bug you encountered.
```

Comment:
@MeikelLP can you maybe copy a good or great bug description from a previous issue that we could easily turn into a nice, exemplary placeholder? Same for the `how to reproduce` input.

Reply:
Then we could either follow @uutkukorkmaz 's recommendation (use ChatGPT) , write one ourselves OR open corresponding issues `Enhance Issue Templates with exemplary placeholders` and label them with `help-wanted` + `good first issue`.

Reply:
I think a template wouldn't hurt but it's not mandatory rn.

We don't have users yet. Don't overengineer the support pipeline :D